admin/menu/class-replacevar-editor.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Menu
+ */
+
+/**
+ * Renders a replacement variable editor.
+ */
+class WPSEO_Replacevar_Editor {
+	/**
+	 * @var Yoast_Form  Yoast Forms instance.
+	 */
+	private $yform;
+	/**
+	 * @var string      The id for the hidden title field.
+	 */
+	private $title;
+	/**
+	 * @var string The id for the hidden description field.
+	 */
+	private $description;
+
+	/**
+	 * WPSEO_Replacevar_Editor constructor.
+	 *
+	 * @param Yoast_Form $yform       Yoast forms.
+	 * @param string     $title       The title field id.
+	 * @param string     $description The description field id.
+	 */
+	public function __construct( $yform, $title, $description ) {
+		$this->yform = $yform;
+		$this->title = $title;
+		$this->description = $description;
+	}
+
+	/**
+	 * Renders a div for the react application to mount to, and hidden inputs where
+	 * the app should store it's value so they will be properly saved when the form
+	 * is submitted.
+	 *
+	 * @return void
+	 */
+	public function render() {
+		$this->yform->hidden( $this->title, $this->title );
+		if ( ! is_null( $this->description ) ) {
```

Comment:
If I pass a boolean as description it will try to render a field. I would check if it is a string and not empty. 

```! empty( $this->description ) && is_string( $this->description )```